### DISCUSSION TITLE
GraphQL subscriptions

### DISCUSSION THREAD

**Discussion by dushaniw** [Kong/kong#8638] [Help]
Does Kong gateway supports GraphQL subscriptions over websocket or SSE?

**Comment by StarlightIbuki** [comment]
Kong(and  Nginx) do support WebSocket and SSE. So, it should work (if you properly set up the proxy to the upstream).
But I am not sure whether GarphQL related plugins support that. (They are using GraphQL over HTTP in their examples).
